www/src/components/homepage/homepage-newsletter.js

```diff
@@ -11,100 +12,90 @@ import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
-        ${p => p.theme.colors.newsletter.stripeColorB} 40px,
-        ${p => p.theme.colors.newsletter.stripeColorB} 60px,
+        ${t.colors.newsletter.stripeColorB} 40px,
+        ${t.colors.newsletter.stripeColorB} 60px,
         transparent 60px,
         transparent 80px
-      );
-    bottom: 0;
-    content: "";
-    height: ${p => p.theme.space[stripedBorderHeight]};
-    left: 0;
-    right: 0;
-    position: absolute;
-  }
+      )`,
+    bottom: 0,
+    content: `""`,
+    height: t => t.space[stripedBorderHeight],
+    left: 0,
+    right: 0,
+    position: `absolute`,
+  },
 
-  ${mediaQueries.lg} {
-    flex-direction: row;
-    justify-content: space-between;
+  [mediaQueries.lg]: {
+    flexDirection: `row`,
+    justifyContent: `space-between`,
 
-    > * {
-      flex-basis: 50%;
-    }
-  }
-`
+    "> *": {
+      flexBasis: `50%`,
+    },
+  },
+}
 
-const Ornament = styled(`span`)`
-  left: -${p => p.theme.space[1]};
-  position: absolute;
-  top: -${p => p.theme.space[2]};
-`
+const ornamentStyles = {
+  left: -1,
+  position: `absolute`,
+  top: -2,
+}
 
-const Name = styled(`h3`)`
-  color: ${p => p.theme.colors.textMuted};
-  font-family: ${p => p.theme.fonts.heading};
-  font-size: ${p => p.theme.fontSizes[1]};
-  font-weight: ${p => p.theme.fontWeights.body};
-  letter-spacing: ${p => p.theme.letterSpacings.tracked};
-  margin: 0;
-  text-transform: uppercase;
-`
+const nameStyles = {
+  color: `textMuted`,
+  fontFamily: `heading`,
+  fontSize: 1,
+  fontWeight: `body`,
+  letterSpacing: `tracked`,
+  m: 0,
+  textTransform: `uppercase`,
+}
 
-const Title = styled(`h1`)`
-  color: ${p => p.theme.colors.newsletter.heading};
-  font-size: ${p => p.theme.fontSizes[4]};
-  font-weight: ${p => p.theme.fontWeights.heading};
-  line-height: ${p => p.theme.lineHeights.dense};
-  margin: 0;
-  margin-top: ${p => p.theme.space[1]};
-`
-
-const Form = styled(EmailCaptureForm)`
-  margin-top: ${p => p.theme.space[5]};
-
-  ${mediaQueries.lg} {
-    margin-top: 0;
-  }
-`
+const titleStyles = {
+  color: `newsletter.heading`,
+  fontSize: 4,
+  fontWeight: `heading`,
+  lineHeight: `dense`,
+  m: 0,
+  mt: 1,
+}
 
 const HomepageNewsletter = () => (
   <HomepageSection>
-    <Container>
-      <Ornament>
+    <Flex sx={containerStyles}>
+      <span sx={ornamentStyles}>
         <NewsletterFormOrnament />
-      </Ornament>
+      </span>
       <header>
-        <Name>The Gatsby Newsletter</Name>
-        <Title>Keep up with the latest things Gatsby!</Title>
+        <h3 sx={nameStyles}>The Gatsby Newsletter</h3>
+        <h1 sx={titleStyles}>Keep up with the latest things Gatsby!</h1>
       </header>
-      <Form
+      <EmailCaptureForm
         isHomepage={true}
         confirmMessage="Success! You have been subscribed to the Gatsby newsletter. Expect to see a newsletter in your inbox each Wednesday (or the equivalent of US Wednesday in your time zone)!"
+        sx={{ mt: [5, null, null, null, 0] }}
```

Comment:
We should check that this actually compiles correctly. `EmailCaptureForm` isn't a raw tag so I'm not sure if the classes will propagate.

Reply:
Threw in some color, LGTM?!

![image](https://user-images.githubusercontent.com/21834/85934785-5e2aad00-b8e8-11ea-8a8c-a757a969d95a.png)